nginx/nginx.conf

```diff
@@ -1,4 +1,4 @@
-user                    <%= nginxConf.NGINX_USER %> <%= nginxConf.NGINX_GROUP %>;
+user                    jimmykasprzak staff;
```

Comment:
Hello, I think this config file was included in the pull request by accident, could you take it out please? :)